src/CMakeLists.txt

```diff
@@ -226,6 +232,15 @@ if(Omega_h_USE_SimModSuite)
   target_link_libraries(omega_h PUBLIC "${SIMMODSUITE_LIBS}")
 endif()
 
+if(Omega_h_USE_ADIOS2)
+  if (Omega_h_USE_MPI)
+    target_include_directories(omega_h PRIVATE "${ADIOS2_INCLUDE_DIRS}")
+    target_link_libraries(omega_h PUBLIC "${ADIOS2_LIBRARIES}")
```

Comment:
@cwsmith @seegyoung  should we be using the adios2 target here? Or, do we need to handle this with the library variables because of `bob.cmake`?

Reply:
I think Cameron can answer this